### PR TITLE
feat: update helm chart and add zgw endpoints

### PR DESCRIPTION
**Pull request by philipsens**
I updated the Helm Chart to match the Helm of the Frank!Framework (ibissource/charts) and added a ConfigMap with the ZGW endpoints.

Let me know if I'm missing anything.

### CODE REVIEW

File: charts/zaakbrug/values.yaml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      zaaktype:
+      rol:
+      roltype:
+      statustype:
+      status:
+      resultaattype:
+      resultaat:
+      informatieobjecttype:
+      enkelvoudiginformatieobjecten:
+      zaakinformatieobjecten:
```
Comment (MLenterman) [inline charts/zaakbrug/values.yaml:113]: Fill in the defaults(from configmap defaults)

File: charts/zaakbrug/values.yaml [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  credentials: 
+    secret:
+    # Key of secret and place to mount.
+    key:
+  # (required) the name of the Frank
+  instance:
+    name: "ZaakBrug"
+  dtap:  
+    # (required) Options: LOC, DEV, TST, ACC, PRD
+    stage:
```
Comment (MLenterman) [inline charts/zaakbrug/values.yaml:126]: Default DEV?

**Review by MLenterman — CHANGES_REQUESTED** [review]

File: charts/zaakbrug/values.yaml (hunk 1 above)
Comment (philipsens) [reply charts/zaakbrug/values.yaml:113]: Should the defaults be at both places or just in the values?

File: charts/zaakbrug/values.yaml (hunk 2 above)
Comment (philipsens) [reply charts/zaakbrug/values.yaml:126]: I'm not sure. The thought process of leaving it empty is as follows:
A Frank! normally won't start if you haven't set the instance.name and dtap.stage. 
I thought to replicate this functionality by leaving it empty and to give a warning while installing the helm.
It forces the user to think about wich dtap stage should be used.

File: charts/zaakbrug/values.yaml (hunk 1 above)
Comment (MLenterman) [reply charts/zaakbrug/values.yaml:113]: I would say just in values.yml should be enough.

**Review by MLenterman — APPROVED** [review]

**Pull request merged**